src/argilla/server/schemas/v1/fields.py

```diff
@@ -38,3 +38,12 @@ class Field(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class FieldSettingsUpdate(BaseModel):
+    use_markdown: Optional[bool]
+
+
+class FieldUpdate(BaseModel):
+    title: Optional[str]
+    settings: Optional[FieldSettingsUpdate]
```

Comment:
Just to align with other definitions
```suggestion
class TextFieldSettingsUpdate(BaseModel):
    use_markdown: Optional[bool]


class FieldUpdate(BaseModel):
    title: Optional[str]
    settings: Optional[TextFieldSettingsUpdate]
```